sufia-models/app/services/sufia/full_text_extraction_service.rb

```diff
@@ -0,0 +1,37 @@
+module Sufia
+  class FullTextExtractionService
```

Comment:
@jcoyne The other services modify the generic_file internally.  This one just returns the extracted text.  A comment about what the service does might be helpful in the future.